lib/kdump_utils.pm

```diff
@@ -113,7 +113,13 @@ sub activate_kdump {
 
 sub activate_kdump_without_yast {
     # activate kdump by grub, need a reboot to start kdump
-    my $cmd = "if [ -e /etc/default/grub ]; then sed -i '/^GRUB_CMDLINE_LINUX_DEFAULT/ s/\"\$/ crashkernel=256M,high crashkernel=128M,low \"/' /etc/default/grub; fi";
+    my $cmd = "";
+    if (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'aarch64')) {
+        $cmd = "if [ -e /etc/default/grub ]; then sed -i '/^GRUB_CMDLINE_LINUX_DEFAULT/ s/\"\$/ crashkernel=256M \"/' /etc/default/grub; fi";
```

Comment:
I'm not sure if you need to specify a memory offset with something like `crashkernel=256M@12345`. I doubt it, so I have merged the PR and if it fails you may need to pick a suitable memory offset.